packages/hawtio/src/plugins/camel/routes/Source.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { jolokiaService } from '@hawtiosrc/plugins'
+import { CodeEditor, Language } from '@patternfly/react-code-editor'
+import { isRoutesFolder } from '@hawtiosrc/plugins/camel/camel-content-service'
+import { CamelContext } from '@hawtiosrc/plugins/camel/context'
+import { MBeanNode } from '@hawtiosrc/plugins/shared'
+
+export const findParentContext = (child: MBeanNode, root: MBeanNode): MBeanNode | undefined => {
```

Comment:
why export this? is it supposed to be used anywhere else?

Reply:
I had it in the different file. Thanks for notify.

Reply:
Is it still relevant?